driver/razerkbd_driver.c

```diff
@@ -653,6 +653,14 @@ static ssize_t razer_attr_read_tartarus_profile_led_red(struct device *dev, stru
     struct usb_interface *intf = to_usb_interface(dev->parent);
     struct usb_device *usb_dev = interface_to_usbdev(intf);
     struct razer_report report = razer_chroma_standard_get_led_state(VARSTORE, RED_PROFILE_LED);
+
+    switch(usb_dev->descriptor.idProduct) {
+    case USB_DEVICE_ID_RAZER_ORBWEAVER_CHROMA:
+    case USB_DEVICE_ID_RAZER_TARTARUS_CHROMA:
+        report = razer_chroma_standard_get_led_state(VARSTORE, ALTERNATE_RED_PROFILE_LED);
+        break;
+    }
+
     struct razer_report response = razer_send_payload(usb_dev, &report);
```

Comment:
Move the declaration of `response` to before the switch. It should fix the CI errors.